### PR TITLE
mapping gender fields from pds excel to EPS values

### PR DESCRIPTION
**Pull request by callumtipper-IW**

### CODE REVIEW

File: client/index.html [hunk 1]
```diff
@@ -414,13 +414,32 @@ <h3 class="nhsuk-panel-with-label__label">Signature</h3>
...
-              <div rv-each-prescription="sendBulkResponse.prescriptions" class="nhsuk-summary-list__row">
-                  <div style="float:left;">{prescription.id}</div>
-              </div>
-            </dl>
-          </div>
+          <button class="nhsuk-button" type="submit" onclick="copySuccessfulPrescriptionIds();" rv-show="clipboardAvailable">
+              Copy Successful Prescription IDs
+          </button>
+          <table class="nhsuk-table">
+              <caption class="nhsuk-table__caption">Bulk send results</caption>
```
Comment (dan-jones-iw) [inline client/index.html:421]: remove caption

File: client/index.ts [hunk 2]
```diff
@@ -145,6 +141,17 @@ customWindow.sendPrescriptionRequest = function () {
   }
 }
 
+customWindow.copySuccessfulPrescriptionIds = function () {
+  if (pageData.sendBulkResponse) {
```
Comment (dan-jones-iw) [inline client/index.ts:145]: remove

File: server/routes.py [hunk 3]
```diff
@@ -339,27 +348,72 @@ def get_send():
...
     }
 
 
+def post_send_bulk():
+    session_short_prescription_ids = get_all_prescription_ids_from_cookie()
+    access_token = get_access_token()
+
+    send_requests = [
+        (short_prescription_id, load_send_request(short_prescription_id))
+        for short_prescription_id in session_short_prescription_ids
```
Comment (dan-jones-iw) [inline server/routes.py:394]: can probably refactor most of this stuff down into one loop over `session_short_prescription_ids`

Comment (mfjarvis) [reply server/routes.py]: Yep

File: server/routes.py [hunk 4]
```diff
@@ -339,27 +348,72 @@ def get_send():
...
-                access_token, request, request_id
-            )
+    if len(session_short_prescription_ids) == 1:
+        return post_send_single()
+    else:
+        return post_send_bulk()
+
+
+def post_send_single():
+    session_short_prescription_ids = get_all_prescription_ids_from_cookie()
```
[the remark this answers is not in the document]
Comment (mfjarvis) [reply server/routes.py]: Thought it was neater :shrug: changed

File: client/index.html (hunk 1 above)
Comment (mfjarvis) [reply client/index.html]: Recommended to have one for accessibility but I guess we're not bothering with that anywhere else. Removed

File: client/index.ts (hunk 2 above)
Comment (mfjarvis) [reply client/index.ts]: Removed

**Review by dan-jones-iw — APPROVED** [review]

**Pull request merged**